doc/api/n-api.md

```diff
@@ -5700,6 +5743,32 @@ support it:
 * If the function is not available, provide an alternate implementation
   that does not use the function.
 
+#### `node_api_is_feature_enabled`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+```c
+NAPI_EXTERN napi_status node_api_is_feature_enabled(napi_env env,
+                                                    node_api_features feature,
+                                                    bool* result);
+```
+
+* `[in] env`: The environment that the API is invoked under.
+* `[in] feature`: The feature that we want to test.
```

Comment:
```suggestion
* `[in] feature`: The features that we want to test.
```

Reply:
Changed. Though I am not sure why we need to use plural form, while the parameter is singular. Should I also rename the parameter?